.gitignore

```diff
@@ -19,3 +19,6 @@ status-console-client
 
 .vscode
 .idea
+
+vendor/github.com/status-im/nimbus
```

Comment:
Let's add a comment why we comment this out.

Reply:
Done